### DISCUSSION TITLE
v5.4.0

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16458] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.4.0'>v5.4.0</a>.</em>

**Comment by s-ohnishi** [comment]
In my environment (WSL2+Docker), I was unable to build an image with 5.4.0, which was no problem with 5.3.1.
(Based on `php:8.1-fpm-buster`, install the necessary things and then run `pecl install phalcon-5.4.0`)
I changed the memory allocated to WSL2 from 4GB to 8GB and it was completed successfully.

**Comment by AnastasisDasyras** [comment]
Is there any page we can find the end of life date of each version? I want to know the end of life for the v5.4.0 of Phalcon. Thanks

**Reply by niden** [reply]
https://docs.phalcon.io/latest/releases/

5.4 was the last version we supported PHP 7+ so it is not receiving any updates.
